assets/js/pages/game.test.js

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
```

Comment:
A little confused about why the tests are not in their own directory inside `assets`...sorry, I probably should have taken a closer look before, but I didn't notice until now. Do they not run if you don't have it structured this way?

Reply:
I've seen this structure on a few projects, and it `works`, but IMO it's really not a good structure so I'd rec moving it